packages/backend-core/src/api/collection/UserApi.ts

```diff
@@ -72,11 +73,12 @@ export class UserApi extends AbstractApi {
   }
 
   public async createUser(params: CreateUserParams): Promise<User> {
-    const { publicMetadata, privateMetadata, unsafeMetadata } = params;
+    const { publicMetadata, privateMetadata, unsafeMetadata, skipPasswordChecks = false } = params;
```

Comment:
@chanioxaris Since params already contain skipPasswordChecks in their definition the spread operator will make it work as expected if the passes it. So, let's revert these changes.